sdk/python/jobs/automl-standalone-jobs/automl-forecasting-github-dau/auto-ml-forecasting-github-dau.ipynb

```diff
@@ -814,9 +827,9 @@
   ],
   "hide_code_all_hidden": false,
   "kernelspec": {
-   "display_name": "Python 3.10 - SDK V2",
```

Comment:
Please remove the `Next Step: Load the best model and try predictions` section.

Reply:
Section removed